### PR TITLE
Decision tree birtdate

### PR DESCRIPTION
**Pull request by AlexisMNI**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [x] New feature (non-breaking change which adds functionality)

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** [(Link to the ticket or issue)](https://impulsumvc.atlassian.net/browse/CATCH-138)

**Does this PR introduce a breaking change?** no

**Other information**:

**Storybook Link** [(Link to a local storybook url to review new functionality)](http://localhost:8080/?path=/story/decisiontree-decisiontree--all)

### CODE REVIEW

File: packages/decision-tree/src/components/Question/questionTypes/Date/Date.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    if (!isValid(date)) return;
+
+    const formattedDate: string = format(date, isoDateFormat);
+    saveAnswer({ type: 'date', value: formattedDate });
+  };
+
+  const defaultDate = defaultValue?.value ? format(defaultValue.value, uiDateFormat) : null;
+
+  return (
+    <Box>
```
Comment (tkanzakic) [inline packages/decision-tree/src/components/Question/questionTypes/Date/Date.tsx:22]: nit: can this be a fragment instead?

File: packages/decision-tree/src/components/Question/questionTypes/Date/Date.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      <QuestionTitle question={label} whyWeAskExplanation={whyWeAskExplanation} />
+      <Flex px={inputMargin}>
+        <DateInput
+          key={id}
+          variant="outline"
+          placeholder={placeholder}
+          defaultValue={defaultDate}
+          onInput={handleChange}
+          mode="MM/DD/YYYY"
+          textTransform="uppercase"
```
Comment (tkanzakic) [inline packages/decision-tree/src/components/Question/questionTypes/Date/Date.tsx:32]: is this needed?

File: packages/decision-tree/src/components/Question/questionTypes/Date/Date.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    <Box>
+      <QuestionTitle question={label} whyWeAskExplanation={whyWeAskExplanation} />
+      <Flex px={inputMargin}>
+        <DateInput
+          key={id}
+          variant="outline"
+          placeholder={placeholder}
+          defaultValue={defaultDate}
+          onInput={handleChange}
+          mode="MM/DD/YYYY"
```
Comment (tkanzakic) [inline packages/decision-tree/src/components/Question/questionTypes/Date/Date.tsx:31]: is this the appropriate format, all uppercased?

**Review by tkanzakic — APPROVED** [review]

File: packages/decision-tree/src/components/Question/questionTypes/Date/Date.tsx (hunk 2 above)
Comment (AlexisMNI) [reply packages/decision-tree/src/components/Question/questionTypes/Date/Date.tsx:32]: Yes, design is in uppercase, and the inputdate component does not allow me to modify the text while typing, so it was very weird having the palceholder uppercase, and then while typing in lowercase. If you can thing of a different solution, please let me know

File: packages/decision-tree/src/components/Question/questionTypes/Date/Date.tsx (hunk 1 above)
Comment (AlexisMNI) [reply packages/decision-tree/src/components/Question/questionTypes/Date/Date.tsx:22]: no, it breaks everything, we need the box display and margins

**Pull request merged**